lib/pages/search/search.dart

```diff
@@ -29,19 +30,24 @@ class Search extends StatefulWidget {
 }
 
 class SearchController extends State<Search>
-    with ComparablePresentationSearchMixin, LoadMoreSearchMixin {
+    with ComparablePresentationSearchMixin {
   static const int limitPrefetchedRecentChats = 3;
   static const int limitSearchingPrefetchedRecentContacts = 30;
   static const int limitPrefetchedRecentContacts = 5;
 
   SearchContactsAndChatsController? searchContactAndRecentChatController;
+
+  final ScrollController mainScrollController = ScrollController();
```

Comment:
you haven't use it anywhere

Reply:
- IMO: I just keep old logic !

<img width="603" alt="Screenshot 2023-11-10 at 09 08 51" src="https://github.com/linagora/twake-on-matrix/assets/99852347/9de49585-9eb6-4d94-adf1-7d10c6cd917b">

Reply:
feel free to remove it